cli/cmd/upgrade/upgrade.go

```diff
@@ -775,6 +775,7 @@ func (c *upgradeSpec) persistPrometheusDataDuringUpgrade() error {
 	// scale down prometheus replicas to 0
 	fmt.Println("Migrating the underlying prometheus persistent volume to new prometheus instance...")
 	prometheus := root.HelmReleaseName + "-prometheus-server"
+	// FIXME(paulfantom): Test if this even working as Prometheus is not deployed with Deployment object but with CRD.
```

Comment:
I do see that `//FIXME` can be useful but it can easily get out of control (eg. become heavily used). If some part of a code needs fixing I'd rather create an issue for that. Otherwise things like this might just remain in the code for indefinite time (if I am correct we don't have any mechanism (eg linter) to check for FIXME/TODO which makes it more likely that stuff might never get fixed).

Reply:
I agree that this requires an issue to follow up, but such issues should be open after PR is merged to prevent creation of stale issues. Bear in mind that this PR is a draft.